docs/index.rst

```diff
@@ -0,0 +1,27 @@
+.. google-auth documentation master file, created by
+   sphinx-quickstart on Thu Sep 22 12:50:15 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
```

Comment:
Also `sphinx-apidoc` seems a much more future proof way of maintaining docs than having hand-crafted prose **outside** of Python files